cv32e40p_manifest.flist

```diff
@@ -27,12 +27,10 @@
 +incdir+${DESIGN_RTL_DIR}/../bhv
 +incdir+${DESIGN_RTL_DIR}/../bhv/include
 +incdir+${DESIGN_RTL_DIR}/../sva
-+incdir+${DESIGN_RTL_DIR}/vendor/pulp_platform_common_cells/include
 
 ${DESIGN_RTL_DIR}/include/cv32e40p_apu_core_pkg.sv
 ${DESIGN_RTL_DIR}/include/cv32e40p_fpu_pkg.sv
 ${DESIGN_RTL_DIR}/include/cv32e40p_pkg.sv
-${DESIGN_RTL_DIR}/../bhv/include/cv32e40p_tracer_pkg.sv
```

Comment:
why did we delete the tracer_pkg?

Reply:
I didn't remove it, just moved it at the bottom before tb_wrapper to have clear separation between CV32 IP and behavioural/test-bench stuff.